src/main/kotlin/link/infra/packwiz/installer/UpdateManager.kt

```diff
@@ -97,6 +99,105 @@ class UpdateManager internal constructor(private val opts: Options, val ui: IUse
 			handleCancellation()
 		}
 
+		// MultiMC MC and loader version checker
```

Comment:
Could you move this to a separate class/file? Should make it easier to merge with some of the other pending changes I've got

Reply:
Just stick it in the `installer` package for now, I can refactor it later. Returning a bool or enum would work, ideally only access the cancellation code from `UpdateManager` itself. `LauncherUtils` would be good, particularly if I want to add support for other launchers in the future.

Reply:
It's what imma do for now, if you want me to change it, comment on this and if not you can just mark it as resolved and merge the PR.

Reply:
It's what imma do for now, if you want me to change it, comment on this and if not you can just mark it as resolved and merge the PR.